ci-operator/step-registry/aws/upi-templates-generate/aws-upi-templates-generate-commands.sh

```diff
@@ -0,0 +1,2780 @@
+#!/bin/bash
+
+# 01_vpc.yaml
+# 01_vpc_disconnected.yaml
+# 02_cluster_infra.yaml
+# 03_cluster_security.yaml
+# 04_cluster_bootstrap.yaml
+# 05_cluster_master_nodes.yaml
+# 06_cluster_worker_node.yaml
+# 97_apps_ingress-elb_dns.yaml
+# 99_int_service_instance.yaml
+
+#
+# template: 01_vpc.yaml
+# 
+echo "Creating 01_vpc.yaml"
+cat > "${SHARED_DIR}/01_vpc.yaml" <<EOF
```

Comment:
These feel a lot like [the installer UPI templates][1].  If they're different, it would be nice to have the step docs explain how, and also why we decided to put the different manifests in a step instead of alongside the existing installer templates.  If they're the same, it's not clear what we gain by duplicating them in a new location.

[1]: https://github.com/openshift/installer/tree/master/upi/aws/cloudformation

Reply:
Updated. All templates are generated in step `upi-install-aws-cluster-custom` locally now and removed VPC parts.